### PR TITLE
Showcase for the Arithmetics Language

### PR DESCRIPTION
**Pull request by emilkrebs**
~~This PR is needed first because I changed the language server: https://github.com/langium/langium/pull/1013/files~~
The PR got merged, so this PR is ready to go.

arithmetics

**New**

- added showcase for the arithmetics language

**Fixed**

- bug when having warnings inside of the code
- bad responsiveness on smaller screen sizes 

**Honorable mention**: Another banner for the arithmetics language is still needed

### CODE REVIEW

**Comment by spoenemann** [comment]
Could you find a better example calculation to show when the page is loaded? The "example1" module is nonsense and does not represent a good showcase (we should change this in the langium main repo, too).

A complex calculation that can be represented with the basic operators would be good, ideally also with a few helper functions and intermediate values / constants.

**Comment by emilkrebs** [comment]
> Could you find a better example calculation to show when the page is loaded? The "example1" module is nonsense and does not represent a good showcase (we should change this in the langium main repo, too).
> 
> A complex calculation that can be represented with the basic operators would be good, ideally also with a few helper functions and intermediate values / constants.

Definitely! I will work on a good representation/example for the arithmetics language!

**Review by spoenemann — COMMENTED** [review]
Excellent work @emilkrebs!

Two final things before merging:
 - We need another preview image on the Showcase page. Would you like to create one yourself in the style of the state machine example, or rather wait until @KatharinaBrinkhaus is back?
 - The Price Calculator should be formatted with shorter line, so they fit better into the page without scrolling.

**Comment by emilkrebs** [comment]
Thanks @spoenemann!
I've added a banner for the arithmetics language and formatted the code of the Price Calculator.

screenshot

File: hugo/content/showcase/arithmetics.html
Comment (spoenemann) [inline hugo/content/showcase/arithmetics.html:9]:
```suggestion
description: A language for arithmetic calculations. It supports definitions of functions and intermediate values.
```

**Review by spoenemann — APPROVED** [review]
One last remark, then let's merge it 👍 

Please remember to *squash* when merging (click the green drop-down icon at the merge button > "Squash and merge").

**Comment by msujew** [comment]
@emilkrebs You forgot to address @spoenemann's last comment.

**Comment by msujew** [comment]
There is an [unresolved conversation](https://github.com/langium/langium-website/pull/143#discussion_r1203481940) right above my last comment.

**Comment by emilkrebs** [comment]
@msujew aah, sorry. I will resolve it and open a PR later that day.

**Pull request merged**
